### PR TITLE
chore: update devtools patches

### PR DESCRIPTION
**Pull request by andyjessop**
## What this PR solves / how to test

Fixes #[insert GH or internal issue number(s)].

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [ ] Tests included
  - [ ] Tests not necessary because:
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [ ] Required
  - [ ] Not required because:
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [ ] Changeset included
  - [ ] Changeset not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [ ] Documentation not necessary because:

### CODE REVIEW

File: packages/wrangler-devtools/.gclient
Comment (penalosa) [inline packages/wrangler-devtools/.gclient:1]: This should be added to the gitignore

File: packages/wrangler-devtools/.gclient_entries
Comment (penalosa) [inline packages/wrangler-devtools/.gclient_entries:1]: As above

File: packages/wrangler-devtools/.gclient_previous_sync_commits
Comment (penalosa) [inline packages/wrangler-devtools/.gclient_previous_sync_commits:1]: As above

File: packages/wrangler-devtools/.gcs_entries
Comment (penalosa) [inline packages/wrangler-devtools/.gcs_entries:1]: As above

File: packages/wrangler-devtools/Makefile
Comment (penalosa) [inline packages/wrangler-devtools/Makefile:1]: 🎉

File: packages/wrangler-devtools/patches/0001-Support-viewing-files-over-the-network.patch [hunk 1]
```diff
@@ -74,16 +74,16 @@ index a0c126df12..4b8244ea69 100644
    persistence: UI.ViewManager.ViewPersistence.PERMANENT,
    async loadView() {
      const Sources = await loadSourcesModule();
--    return Sources.SourcesNavigator.FilesNavigatorView.instance();
+-    return new Sources.SourcesNavigator.FilesNavigatorView();
 +    return Sources.SourcesNavigator.NetworkNavigatorView.instance();
```
Comment (penalosa) [inline packages/wrangler-devtools/patches/0001-Support-viewing-files-over-the-network.patch:78]: Does this need a `new`?

File: packages/wrangler-devtools/patches/0003-Support-previewing-subrequest-responses.patch [hunk 2]
```diff
@@ -22,42 +22,42 @@ index c8aa624961..6201d73bba 100644
...
        networkRequest = this.maybeAdoptMainResourceRequest(requestId);
-@@ -714,6 +714,11 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
+@@ -803,6 +803,11 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
        return;
      }
      this.getExtraInfoBuilder(requestId).finished();
 +    if (cfResponse !== undefined) {
 +      networkRequest.setContentDataProvider(async () => {
-+        return { error: null, content: cfResponse.body, encoded: cfResponse.base64Encoded };
++        return { error: '', content: cfResponse.body, encoded: cfResponse.base64Encoded };
```
Comment (penalosa) [inline packages/wrangler-devtools/patches/0003-Support-previewing-subrequest-responses.patch:31]: Why this change?

File: packages/wrangler-devtools/patches/0001-Support-viewing-files-over-the-network.patch (hunk 1 above)
Comment (andyjessop) [reply packages/wrangler-devtools/patches/0001-Support-viewing-files-over-the-network.patch]: Yep, good spot 👍

File: packages/wrangler-devtools/patches/0003-Support-previewing-subrequest-responses.patch (hunk 2 above)
Comment (andyjessop) [reply packages/wrangler-devtools/patches/0003-Support-previewing-subrequest-responses.patch:31]: This is a new type constraint in the frontend-devtools application.

File: packages/wrangler-devtools/patches/0001-Support-viewing-files-over-the-network.patch (hunk 1 above)
Comment (andyjessop) [reply packages/wrangler-devtools/patches/0001-Support-viewing-files-over-the-network.patch]: Actually, apparently not. It uses the static method to instantiate.

**Comment by penalosa** [comment]
Closing in favour of https://github.com/cloudflare/workers-sdk/pull/7137
